[2024-02-26 15:34:04] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-02-26 15:34:09] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-02-26 15:34:14] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-02-26 15:34:19] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-02-26 15:34:21] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 47bf17240c1a78aa 127.0.0.1:63133 failed checkpoint posthandshake: Connected to self
[2024-02-26 15:34:21] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 47bf17240c1a78aa 127.0.0.1:20134 failed checkpoint posthandshake: Connected to self
[2024-02-26 15:34:24] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-02-26 15:34:29] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-02-26 15:34:34] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-02-26 15:34:39] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-02-26 15:34:44] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-02-26 15:34:49] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-02-26 15:34:54] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-02-26 15:34:54] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 47bf17240c1a78aa 127.0.0.1:63248 failed checkpoint posthandshake: Connected to self
[2024-02-26 15:34:54] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn dyn dial conn 47bf17240c1a78aa 127.0.0.1:20134 failed checkpoint posthandshake: Connected to self
[2024-02-26 15:34:59] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
